activerecord/test/cases/create_new_entry.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+require "cases/helper"
+require "active_record"
+require "minitest/autorun"
+require "logger"
+
+ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
+ActiveRecord::Base.logger = Logger.new(STDOUT)
+
+ActiveRecord::Schema.define do
+  create_table :posts, force: true do |t|
+  end
+end
+
+class CreateNewEntryTest < ActiveRecord::TestCase
```

Comment:
I think it would be a bit excessive to have a whole separate file for this particular test case
I'd say we can find a place for this scenario somewhere around existing `create` tests - https://github.com/rails/rails/blob/f5a1075e71b79eb7eb0f4f6323468ef9c7286ee9/activerecord/test/cases/persistence_test.rb#L499

Reply:
Good point! In my modified PR, the test is moved into persistence_test.rb under a new method.